app/routes/_auth+/dev-login.tsx

```diff
@@ -0,0 +1,32 @@
+import  assert  from "assert"
+import { type DataFunctionArgs } from "@remix-run/server-runtime"
+import { login } from "#app/utils/auth.server.ts"
+import { handleNewSession } from "./login.tsx"
+
+export async function loader({ request }: DataFunctionArgs) {
+
+	const session = await login({
```

Comment:
Is this dev-login page deployed to production?

Reply:
Oh just saw this question. Now this is only working on the refactor-scoring branch.